src/components/Scheme/SchemeProposalsPage.tsx

```diff
@@ -214,10 +213,9 @@ const SubscribedSchemeProposalsPage = withSubscription<IProps, SubscriptionData>
     const schemeId = props.scheme.id;
 
     // this query will fetch al data we need before rendering the page, so we avoid hitting the server
-    // with all separate queries for votes and stakes and stuff...
-
-    const prefetchQuery = gql`
-      query prefetchProposalDataForSchemeProposalsPage {
+    // with all separate queries for votes and stakes and rewards...
```

Comment:
Would be great to have a richer comment here about what data is being fetched and what is being subscribed-to, and what role the "separate queries" play in that story.

Reply:
Please check if this helps: https://github.com/daostack/alchemy/blob/01b911ba4ba4be5ec3da0138f7ab84098fdbdf10/docs/fetchingandsubscribingdata.md